h2o-mapreduce-generic/src/main/java/water/hadoop/HiveTokenGenerator.java

```diff
@@ -35,26 +31,23 @@ public HiveOptions(String hiveHost, String hivePrincipal) {
       _host = hiveHost;
       _principal = hivePrincipal;
     }
+
+    private static boolean isPresent(String value) {
+      return value != null && !value.isEmpty();
```

Comment:
To trim or not to trim ? Not sure if whitespaces are welcome here, just an idea.

![1rs9jw](https://user-images.githubusercontent.com/8769110/54436348-a4c7b880-4732-11e9-9cc2-50b3058db6e2.jpg)